driver/src/main/java/org/neo4j/driver/Bookmark.java

```diff
@@ -35,17 +36,39 @@
  */
 public interface Bookmark
 {
+    /**
+     * Returns a string that this bookmark instance identifies.
+     *
+     * @return a string that this bookmark instance identifies.
+     */
+    String value();
+
     /**
      * Returns a read-only set of bookmark strings that this bookmark instance identifies.
+     *
      * @return a read-only set of bookmark strings that this bookmark instance identifies.
      */
+    @Deprecated
     Set<String> values();
 
     /**
-     * Reconstruct bookmark from \bookmarks string values.
+     * Reconstruct bookmark from bookmark string value.
+     *
+     * @param value value obtained from a previous bookmark.
+     * @return A bookmark.
+     */
+    static Bookmark from( String value )
+    {
+        return InternalBookmark.parse( Collections.singleton( value ) );
+    }
+
+    /**
+     * Reconstruct bookmark from bookmarks string values.
+     *
      * @param values values obtained from a previous bookmark.
      * @return A bookmark.
      */
+    @Deprecated
```

Comment:
The only thing affected under "my" control is OGM and that's good. Proper clear API here.